etc/clang-17-toolchain.cmake

```diff
@@ -1,3 +1,8 @@
+# cmake-format: off
```

Comment:
Does cmake-format not like the file?

Reply:
No.

If removed, `--fix` will do this:

```diff
-# etc/clang-17-toolchain.cmake -*- cmake -*-
-# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# etc/clang-17-toolchain.cmake -*- cmake -*- SPDX-License-Identifier: Apache-2.0
+# WITH LLVM-exception
```